lib/crossing.rb

```diff
@@ -34,8 +42,21 @@ def get(bucket, file)
     File.open(file, 'wb') { |f| f.write(content) }
   end
 
+  def get_multiple(bucket, filelist)
+    filelist.each {|file| get(bucket, file)}
+  end
+
+  def put_multiple(bucket, filelist)
+    filelist.each {|file| put(bucket, file)}
+  end
+
   def get_content(bucket, file)
     @s3_client.get_object(bucket: bucket, key: file).body.read
+
+  # If a decryption exception occurs, warn and get the object without decryption
+  rescue Aws::S3::Encryption::Errors::DecryptionError
+    STDERR.puts "WARNING: \""+file+"\" decryption with the key failed. Retreiving the object without encryption."
```

Comment:
You can use string intepolation to make this look a bit cleaner:

`"WARNING: \"#{file}\" decryption with the key failed. Retreiving the object without encryption."`

Reply:
Thanks, updated